packages/store/src/internal/state-factory.ts

```diff
@@ -116,6 +116,8 @@ export class StateFactory {
         defaults: StateFactory.cloneDefaults(meta.defaults)
       };
 
+      meta.instance = stateMap.instance;
```

Comment:
Oh no! This `instance` property was supposed to be removed from the `MetaDataModel` a very long time ago.
It was added by a mistake by Danny in this commit: https://github.com/ngxs/store/commit/35f3e149fe3b266ef1aaea61f7f5f67c7af2cb2a
It is not used and should be removed. I'll add a PR to remove it.
It should not be confused with the `instance` property on the `MappedStore` objects which are maintained within the `StateFactory`.

Having this on the type violates encapsulation so badly. The instance must remain within the objects managed by the Angular Injector.